README.md

```diff
@@ -1,64 +1,159 @@
 
-# ecs-up8r
+# ecs-updater
 
-Runs the whole process of restarting an AWS ECS service with a freshly built Docker image:
-
-1. Login to ECR
-2. Build a Docker image
-3. Tag the image so that Docker knows to upload it to the ECR repository
-4. Push the image to the repository
-5. Register a new ECS taskDefinition with a reference to the newly built image
-6. Restart the ECS service using the new taskDefinition
-7. Upload metadata into an S3 bucket
+Utility for updating and restarting Docker images
```

Comment:
A bit more accurate would be: "A Node.js CLI utility for updating and restarting AWS Elastic Container Service services."